documentation/README.md

```diff
@@ -0,0 +1,12 @@
+# Documentation
+
+This folder include extra documentation about the API beyond the technical OpenAPI v3 API description of the `code` folder
+
+* [MeetingMinutes](MeetingMinutes/)
+  * Check this folder for the original meeting minutes.
+  
+* [API_documentation](API_documentation/)
+  * Check this folder for extra documentation to support the API definition, UML activity diagrams, API overview description, how to use guides, and other documentation related.
+
+* [SupportingDocuments](SupportingDocuments/)
```

Comment:
See comment below ... you are about to delete the folder.

Reply:
recovered file on commit 25cce13